src/main/java/uk/gov/hmcts/reform/sandl.snlevents/controllers/ProblemController.java

```diff
@@ -20,7 +21,13 @@
 
     @GetMapping(path = "", produces = MediaType.APPLICATION_JSON_VALUE)
     public @ResponseBody
-    List<Problem> getProblems() {
+    List<ProblemResponse> getProblems() {
         return problemService.getProblems();
     }
+
+    @GetMapping(path = "by-entity-id", params = "id", produces = MediaType.APPLICATION_JSON_VALUE)
+    public @ResponseBody
```

Comment:
If we use @RestController @ResponseBody is not needed anymore

Reply:
cool, will change